db/migrations/versions/ef1413eb13d9_changed_fields_from_int_to_fk_in_recipe.py

```diff
@@ -0,0 +1,52 @@
+"""changed fields from int to fk in Recipe
+
+Revision ID: ef1413eb13d9
+Revises: abc4a9801f4e
+Create Date: 2023-11-16 21:04:10.432509
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'ef1413eb13d9'
+down_revision: Union[str, None] = 'abc4a9801f4e'
```

Comment:
once you remove the previous two, you have to fix this to point to the latest from mater